src/views/profiles/viewProfile.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import ProfileView from '../../components/profiles/viewProfile';
+import { fetchProfile, editProfile } from '../../redux/actions/Profile.actions';
+
+class ViewProfile extends Component {
+  state = {};
+
+  componentDidMount() {
+    const { fetchProfile } = this.props;
+    fetchProfile(this.state);
+  }
+
+  render() {
+    const { profile: { username, bio, img_url, first_name, last_name } } = this.props;
+    return (
+      <ProfileView
+        username={username}
+        image={img_url}
+        bio={bio}
+        first_name={first_name}
+        lastname={last_name}
+        handleCick={this.handleCick}
+        handleChange={this.handleChange}
+      />
+    );
+  }
+}
+
+ViewProfile.propTypes = {
+  fetchProfile: PropTypes.func.isRequired,
+  profile: PropTypes.shape({}),
+};
+
+ViewProfile.defaultProps = {
+  profile: {}
+};
+
+const mapStateToProps = state => ({
+  profile: state.ProfileReducer.profile
```

Comment:
Hi @a-braham ,
Would you use destructuring of the state here instead of accessing the properties of the state directly, it would improve on the elegance of your code.